src/markdown-pages/automate-workflows/diagnose-problems/index.mdx

```diff
@@ -173,8 +173,8 @@ With your services running in the cloud, log in to New Relic and select **APM**
 
 Yikes! The alerts, high response times, and red-colored indicators suggest things aren't well. Use New Relic to diagnose these issues, which are simultaneously affecting your services:
 
-- [Issue 1: The Warehouse Portal has abnormally high response times](diagnose-problems/high-response-times)
-- [Issue 2: Multiple services are raising error alerts](diagnose-problems/error-alerts)
+- [Issue 1: The Warehouse Portal has abnormally high response times](high-response-times)
+- [Issue 2: Multiple services are raising error alerts](error-alerts)
```

Comment:
Thanks for making these changes! Could we actually use the full relative URL instead of the relative path to the current page? This is a bit prone to breakage in case this page ever moves around.

```suggestion
- [Issue 1: The Warehouse Portal has abnormally high response times](/automate-workflows/diagnose-problems/high-response-times)
- [Issue 2: Multiple services are raising error alerts](/automate-workflows/diagnose-problems/error-alerts)
```

I _think_ those are the right paths, but would love for you to double check. Thank you!!

Reply:
Added the changes and tested locally, everything looks good!

Reply:
Wonderful. Thanks so much for fixing this!